sympy/functions/special/error_functions.py

```diff
@@ -181,7 +181,16 @@ def _eval_rewrite_as_expint(self, z):
         return sqrt(z**2)/z - z*expint(S.Half, z**2)/sqrt(S.Pi)
 
     def _eval_rewrite_as_tractable(self, z):
-        return S.One - _erfs(z)*exp(-z**2)
+        from sympy.series.limits import limit
+        if len(z.free_symbols) == 1:
+            sym = next(iter(z.free_symbols))
+            L = limit(z, sym, S.Infinity)
+            if L is S.Infinity:
+                return S.One - _erfs(z)*exp(-z**2)
+            elif L is S.NegativeInfinity:
+                return S.NegativeOne + _erfs(-z)*exp(-z**2)
```

Comment:
If neither of these cases is true, the function will return None.

Reply:
That's the intent. The return of None is taken as "no change" by the [rewrite](https://github.com/sympy/sympy/blob/master/sympy/core/basic.py#L1580) method of Basic. At finite points, erf is tractable as is.